app/views/idv/verify_info/show.html.erb

```diff
@@ -15,6 +15,24 @@ locals:
   <!-- Needed by form steps wait javascript -->
 </div>
 
+<% if @had_barcode_read_failure %>
+  <%= render AlertComponent.new(
+        type: :warning,
+        class: 'margin-bottom-4',
+        text_tag: 'div',
+      ) do %>
+    <%= t(
+          'doc_auth.headings.capture_scan_warning_html',
+          link: render(
+            FormLinkComponent.new(
+              href: idv_doc_auth_step_path(step: :redo_document_capture),
```

Comment:
Eventually, redo_document_capture will need to be its own controller and this link will need to change. Looks good for now.